module-2/backend/week_three.md

```diff
@@ -10,14 +10,36 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+<p> rails new project_name -any_flags_you_need</p>
 2. What do Models generally inherit from in rails?
+<p>ActiveRecord::Base</p>
 3. What do Controllers generally inherit from in a rails project?
+<p>They can inherit from ActiveRecord::Base or ApplicationController - in the ApplicationController you can create methods that are accessible across all controllers (I think). It also inherits from ActiveRecord so your controllers have access to the AR methods as well.</p>
```

Comment:
A controller will not inherit from ActiveRecord::Base, that is our model. You're controllers will not work if you inherit from that. 

You get the AR methods through the rails app requireing the model files and then using the model class to use AR. 

You can write methods in your ApplicationController that can be accessed in all other controllers becasue generally all controllers will inherit from ApplicationController